src/charts/utilization/utilization-chart.html

```diff
@@ -1,7 +1,7 @@
 <div class="utilization-chart-pf">
-    <span class="title">{{config.title}}</span>
+    <span class="h3">{{config.title}}</span>
     <div class="current-values">
-        <div class="available-count pull-left">
+        <div class="available-count h1 pull-left">
```

Comment:
Doing so would cause the .content h1 style setting to take over and change the line-height and font-size